bundles/org.openhab.binding.deconz/README.md

```diff
@@ -260,6 +278,14 @@ then
 end
 ```
 
+### Thing Actions
+
+```js
+deconzActions = actions.get("deconz", "deconz:lightgroup:00212E040ED9:5");
+retVal = deconzActions.createScene("TestScene");
+deconzActions.storeScene(retVal["newSceneId"]);
```

Comment:
I will need to analyze this. It's a backport (intended as first PR out of many going chronologically through commits and trying to group them), so I did not write the code.

Perhaps it was intended as being extensible so that more could be added to the DTO and action response without breaking the contract (and by that all user rules).

Reply:
My conclusion is still the same. I have added a line to the documentation explaining this (how, not why).

Reply:
Looks really weird and complex for nothing, but OK as this is at least documented.